format/midi/testdata/key-changes.fqtest

```diff
@@ -0,0 +1 @@
+$ /fq -d midi '.. | select(.event=="KeySignature")?.key' test.mid
```

Comment:
Hmm query should  produce some output?

Reply:
Same for notes.fqtst and and tempo.fqtest

Reply:
Apologies - forgot to update the help file with the snake-cased event names. Fixed.

Reply:
No worries! but forgot to add? looks like CamelCase still